### PR TITLE
Work with default OpenShift SCC

### PR DESCRIPTION
**Pull request by jimmidyson**

### CODE REVIEW

**Comment by jimmidyson** [comment]
@iocanel Can you try this out & make sure it doesn't break anything for you? Works for me :) I know it wasn't broken for you before but as long as this doesn't break for you then we can merge - think it's going to be required in later versions & more secure environments.

**Comment by iocanel** [comment]
@jimmidyson: It breaks with:

Error creating: pods "zipkin-mysql-" is forbidden: unable to validate against any security context constraint: [capabilities.add: Invalid value: "SETGID": capability may not be added capabilities.add: Invalid value: "SETGID": capability may not be added]

**Comment by jimmidyson** [comment]
In a vanilla OpenShift install ZK fails to come up too, unable to create data directory, I'll document the SCC requirements & change this to run under it's own `zipkin` service account.

**Comment by jimmidyson** [comment]
ZK requires fabric8 service account & fabric8 SCC? We should update the images to work without that.

**Comment by jimmidyson** [comment]
I've updated the MySQL image to run on vanilla OpenShift so should work ootb without any SCC changes, etc. I also had to update the zookeeper & kafka apps in fabric8-ipaas so need to release those first & update the images in here to take advantage of it.

**Comment by iocanel** [comment]
Thanks @jimmidyson.

File: mysql/pom.xml [hunk 1]
```diff
@@ -86,4 +86,98 @@
         </plugins>
     </build>
 
+    <profiles>
+        <profile>
+            <id>release</id>
```
Comment (iocanel) [inline mysql/pom.xml]: Why do u add this profile? Doesn't the root profile suffice?

Comment (rawlingsj) [inline mysql/pom.xml]: The release profile is used by the Jenkins Pipeline scripts.  It's the same for all the fabric8 projects so it's easy to add things like the gpg signing etc.

Comment (rawlingsj) [inline mysql/pom.xml]: ah I see yeah the release profile is in the root, yeah I doubt this is needed

Comment (jimmidyson) [inline mysql/pom.xml]: Ah sorry - missed it was in the root, will remove.

Comment (jimmidyson) [inline mysql/pom.xml]: Done.

**Comment by jimmidyson** [comment]
@iocanel Updated with latest ipaas release - should all be working now.

**Pull request merged**